misk/src/main/kotlin/misk/web/metadata/webaction/WebActionMetadata.kt

```diff
@@ -69,7 +72,9 @@ data class WebActionMetadata(
     },
     requestType = requestType.toString(),
     returnType = returnType.toString(),
```

Comment:
Could be wrong, but is return type == response type? And I think the return type is already included in the `types` map so the `responseTypes` map would be duplicative

Reply:
Unfortunately not. `returnType` will be set to the literal Kotlin type return by the function that implements the action. For example `misk.web.Response<com.squareup.exemplar.protos.HelloWebResponse>`. For the response type I'm reflecting on the `KType` to determine the classified class, in this case `com.squareup.exemplar.protos.HelloWebResponse`.

Currently `types` only includes the types for the request parameter annotated with `@RequestBody` - it doesn't include types for the response.